README.adoc

```diff
@@ -18,8 +18,8 @@
 //
 image:https://badges.gitter.im/eclipse/microprofile-concurrency.svg[link="https://gitter.im/eclipse/microprofile-concurrency"]
```

Comment:
do we need to rename the gitter chat?

Reply:
probably doesn't hurt

Reply:
Sure, I'm fine with updating that preemptively. Just added a commit fixing it.